scripts/cat/pelts.py

```diff
@@ -270,13 +283,17 @@ def __init__(
                     break
 
                 # convert paras
-                if self.length == "long":
-                    self.cat_sprites["para_adult"] = "para_adult_long0"
-                else:
-                    self.cat_sprites["para_adult"] = "para_adult_short0"
+                if age == "para_adult":
+                    if self.length == "long":
+                        self.cat_sprites["para_adult"] = "para_adult_long0"
+                    else:
+                        self.cat_sprites["para_adult"] = "para_adult_short0"
+                    continue
 
```

Comment:
```suggestion
                if age == "para_adult":
                    if self.length == "long":
                        self.cat_sprites[age] = "para_adult_long0"
                    else:
                        self.cat_sprites[age] = "para_adult_short0"
                    continue
```
Let's make it a little more consistent with the rest of the checks.